src/clojure_tictactoe/cli/input/input_getter.clj

```diff
@@ -0,0 +1,31 @@
+(ns clojure-tictactoe.cli.input.input-getter
+  (:require [clojure.string :as string]
+            [clojure-tictactoe.game.players :as players]))
+
+(defn get-user-input []
+  (let [user-input (string/trim (read-line))]
+    (if (= user-input "quit")
+      (. System exit 0)
+      user-input)))
+
+(defn continue-to-game []
+  (println "Press enter key to continue")
+  (get-user-input)
+  nil)
+
+(defn parse-move-input
+  ([input]
+   (let [formatted-input (read-string input)]
+     (let [valid-numeral (and
```

Comment:
Good stuff on assigning `let` variables to the smaller functions. Keep that up

Reply:
✅